docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 11
+title: Local Datastore (FactDB)
+---
+
+# PDP Local Datastore (FactDB)
+
+FactDB is an embedded datastore to manage larges data volumes inside the PDP.
+
+It allows the PDP to store and query large volumes of data locally,
+reducing the latency and improving the performance of the PDP.
+
+:::info EAP
+This feature is available under Early Access Program.
+Please contact with us on [Slack](https://io.permit.io/slack) for more information.
+:::
+
+:::warning
+This version is available on **PDP v0.7.0** and above.
+Please make sure to upgrade **all PDPs in the environment** to the latest version to use FactDB.
+:::
+
+## Enabling FactDB
+
+The FactDB can be enabled per Permit Environment, allowing PDPs in that environment to use the FactDB.
+Optionally, you can have both legacy PDPs and FactDB enabled PDPs in the same environment.
```

Comment:
This is a confusing way to state that. Should I enable it for the whole environment and then enable others? Why should I enable that?

I would recommend the following outline:
1. By default, Permit environment comes with FactsDB disabled.
2. You can enable the FactsDB in the environment level to support PDPs to run this way
3. When deploying PDP, you can decide if you want them to have local facts enabled.

Reply:
I'll just remove it from this part, as we have a dedicated part for "both FactDB and Legacy". When you enable, it is enabled for all.